tensorflow/core/kernels/reverse_sequence_op.cc

```diff
@@ -76,8 +76,8 @@ void CheckErrors(OpKernelContext* context, int batch_dim, int seq_dim) {
   }
 }
 
-template <>
-void CheckErrors<GPUDevice>(OpKernelContext* context, int batch_dim,
+template <typename Tlen>
```

Comment:
why do you need to template CheckErrors on Tlen?  I don't see you using Tlen anywhere.